test/Microsoft.AspNet.Mvc.DataAnnotations.Test/DataAnnotationsModelValidatorTest.cs

```diff
@@ -284,11 +314,111 @@ public void Validate_IsValidFalse_StringLocalizerReturnsLocalizerErrorMessage()
             // Assert
             var validationResult = result.Single();
             Assert.Equal("", validationResult.MemberName);
-            Assert.Equal("Longueur est invalide", validationResult.Message);
+            Assert.Equal("Longueur est invalide : 4", validationResult.Message);
+        }
+
+        public static TheoryData Validate_AttributesIncludeValues
+        {
+            get
+            {
+                var pattern = "apattern";
+                var length = 5;
+                var regex = "^((?!" + pattern + ").)*$";
+
+                return new TheoryData<ValidationAttribute, string, object[]>
+                {
+                    {
+                        new RegularExpressionAttribute(regex) { ErrorMessage = LocalizationKey },
```

Comment:
Not a fan since 99.9845% of the time a customer uses an `IStringLocalizer`, it'll be with the default `ErrorMessage` for the attribute. See suggestion for the verification below...